src/planscape/martin/migrations/0001_initial.py

```diff
@@ -0,0 +1,12 @@
+from django.db import migrations
+
+from utils.file_utils import read_file
+
+UP_MIGRATION = read_file("martin/sql/martin_get_treatment_plan_prescriptions.sql")
+DOWN_MIGRATION = "DROP FUNCTION martin_get_treatment_plan_prescriptions;"
```

Comment:
Is it worth adding IF EXISTS here? I know it's unlikely this will be called, but if there was an issue creating this, or if the DB is otherwise weird, this might save us some trouble in the future.